BoxContentSDK/BoxContentSDK/Clients/BOXContentClient+Metadata.h

```diff
@@ -0,0 +1,161 @@
+//
+//  BOXContentClient+Metadata.h
+//  BoxContentSDK
+//
+//  Created on 6/12/15.
+//  Copyright (c) 2015 Box. All rights reserved.
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+#import "BOXMetadataRequest.h"
+
+@class BOXMetadataRequest;
+@class BOXMetadataDeleteRequest;
+@class BOXMetadataCreateRequest;
+@class BOXMetadataUpdateRequest;
+@class BOXMetadataTemplateRequest;
+
+/**
+ * BOXContentClient category that provides convenience methods to setup metadata requests.
+ *
+ * For more information @see BOXMetadataRequest, BOXMetadataDeleteRequest, BOXMetadataCreateRequest, 
+ *                           BOXMetadataUpdateRequest, BOXMetadataTemplateRequest
+ */
+@interface BOXContentClient (Metadata)
+
+/**
+ * Generate a request to retrieve metadata information about a file.
+ *
+ * **NOTE** Defaults the scope property to @see BOXAPIScopeEnterprise.
+ *
+ * @param fileID File ID.
+ * @param template Template for metadata in file.
+ *
+ * @return A request that can be customized and then executed.
+ */
+- (BOXMetadataRequest *)metadataInfoRequestWithFileID:(NSString *)fileID template:(NSString *)template;
```

Comment:
this API does not exist. /files/id/metadata/properties is a one off and is an alias for scope=global template=properties.

I would rather it not be included and force users to be explicit.